<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gaiaViewBase</class>
 <widget class="QWidget" name="gaiaViewBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>gaia_base</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTextEdit" name="textEdit_input"/>
     <widget class="QWebView" name="textEdit_output"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
